packages/react-label/Spec.md

```diff
@@ -6,58 +6,175 @@ _Description and use cases of this component_
+
+Props used in v8 Label:
+|Prop|Description|
+|---|---|
+|as|Element type to render Label as|
+|componentRef|Optional callback to access the ILabel interface. Use this instead of ref for accessing the public methods and properties of the component|
+|disabled|Renders label as disabled, changing the Label's foreground color|
+|required|Renders an asterisk next to the given text|
+|styles|Custom styles for the Label|
+|theme|Theme provided by HOC|
+
+### Label in Northstar/v0
+
+```tsx
+<Label content="You have 23 emails" />
+```
+
+Props used in v0 Label:
+|Prop|Description|
+|---|---|
+|accessibility| Prop to override accessibility behavior|
+|as|Element type to render Label as|
+|circular|Renders Label with round corners|
+|className|Additional CSS class name to apply|
+|color|Color for the background of the label|
+|content|Label content|
+|fluid|Make it so Label takes the width of its container|
+|icon|Adds an icon to the label|
+|iconPosition|Choose where the icon is placed (Start or End)|
+|image|Adds an image to the label|
+|imagePosition|Choose where the image is placed (Start or End)|
+|styles|Additional custom styles for the component|
+|variables|Allows override of theme variables|
+
+### Conclusion
+
+Moving forward, Label will be a simple component. The converged Label will be based on v8's Label with some minor changes. Most props from v0 will be left out but because of the use of slots, it won't be too difficult to migrate to the converged Label.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+The Label component should be simple as shown below. It will just need the text to be rendered. For the required label, it has the option of being a shorthand slot that will allow to customize the label's required text.
+
+```tsx
+<Label>Label</Label>
+
+<Label disabled>Label</Label>
+
+<Label required>Label</Label>
+
+<Label required="**">
+  Label
+</Label>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- A Label can be rendered with an asterisk or custom text when is set as `required`.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+```ts
+/**
+ * Label Props
+ * {@docCategory Label}
+ */
+export interface LabelProps extends ComponentProps, React.LabelHTMLAttributes<HTMLElement> {
+  /**
+   * Renders the label as disabled
+   * @defaultvalue false
+   */
+  disabled?: boolean;
+
+  /**
+   * Whether the associated form field is required or not. If true it will be an asterisk, otherwise it will be what is provided.
+   * @defaultvalue false
+   */
+  required?: boolean | ShorthandProps<ComponentProps>;
+
+  /**
+   * A label supports different sizes.
+   * @defaultvalue 'medium'
+   */
+  size?: 'small' | 'medium' | 'large';
+
+  /**
+   * A label supports semibold/strong fontweight.
```

Comment:
Has the design default changed here? In v8 labels are semibold (or bold?) by default.

Reply:
Yes, based on the design spec regular is now the default.